lib/src/native/realm_core.dart

```diff
@@ -186,6 +186,15 @@ class _RealmCore {
     return RealmHandle._(realmPtr);
   }
 
+  bool deleteRealmFiles(String path) {
```

Comment:
```suggestion
  void deleteRealmFiles(String path) {
```

Reply:
Fixed in https://github.com/realm/realm-dart/pull/169/commits/0697dce4c21edbd697003715971e4e21b9d68d23